packages/web-components/src/components/va-link-action/va-link-action.scss

```diff
@@ -0,0 +1,171 @@
+@forward 'settings';
+@import '../../mixins/links.css';
+@import '../../mixins/accessibility.css';
+@import '~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables.scss';
```

Comment:
We import [variables.css globally](https://github.com/department-of-veterans-affairs/component-library/blob/12e91c223ece239a8f0ce13a6a32203a997c9a23/packages/web-components/src/global/main.css#L1) and prefer to use custom properties (which are supported natively) instead of Sass where possible.

So this import can be removed and the media query can be changed from `$medium-screen` to `var(--medium-screen)`.

Reply:
Hmm. I tried that and it doesn't work. Thoughts? The only other media query that is using a variable is in va-table-inner and I can't confirm if that's working either.

Reply:
Are coding conventions like this one written down anywhere that I can reference?

Reply:
Ah, you're right the custom properties won't work in a media query so importing the Sass variables seems to be the right way forward here. Thanks for calling that out! 

As for conventions, I think we need to revisit our [contributing guidelines](https://design.va.gov/about/developers/contributing). Especially as more external teams are contributing to the system too.

Reply:
PR to update the documentation site with Sass breakpoint variables:

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/4011